PicoPagesImages.php

```diff
@@ -94,15 +102,25 @@ private function getImages()
             }
             list($width, $height,, $size) = array_pad($imagesize, 4, '');
 
+            // Find meta files for images if they exist
+            $metapath = $path . '.meta.yml';
+            if (is_file($metapath)) {
+                $meta = Yaml::parse(file_get_contents($metapath));
+            }
+
             $data[] = array (
                 'url' => $this->getBaseUrl() . $images_path . pathinfo($path, PATHINFO_BASENAME),
                 'path' => $images_path,
                 'name' => pathinfo($path, PATHINFO_FILENAME),
                 'ext' => pathinfo($path, PATHINFO_EXTENSION),
                 'width' => $width,
                 'height' => $height,
-                'size' => $size
+                'size' => $size,
+                'metatitle' => isset($meta['title']) ? $meta['title'] : '',
+                'metadesc' => isset($meta['description']) ? $meta['description'] : ''
```

Comment:
Is there any reason not to return entirely `'meta' => $meta` ?

Reply:
True, I was thinking in terms of limiting the depth of keys available to twig and hadn't considered that. We can leave it as $meta and have them picked through twig, works out cleaner, more flexible.

Reply:
Agree 👍